.github/workflows/ci.yml

```diff
@@ -11,14 +11,15 @@ jobs:
     steps:
       - name: Clone This Repo
         uses: actions/checkout@v2
-      - name: Checkout CMock submodule
-        run: git submodule update --checkout --init --recursive test/unit-test/CMock
+      - name: Checkout Unity submodule
+        run: git submodule update --checkout --init --recursive test/unit-test/Unity
       - name: Build
         run: |
           sudo apt-get install -y lcov
           cmake -S test -B build/ \
           -G "Unix Makefiles" \
           -DCMAKE_BUILD_TYPE=Debug \
+          -DBUILD_CLONE_SUBMODULES=ON \
```

Comment:
This option and the `Checkout Unity submodule` step are redundant. You can use either of them